tests/v2/integration/steveapi/steve_api_test.go

```diff
@@ -1533,8 +1552,10 @@ func (s *SteveAPITestSuite) TestList() {
 			jsonResp, err := formatJSON(secretList)
 			require.NoError(s.T(), err)
 			jsonFilePath := filepath.Join(jsonDir, getFileName(test.user, test.namespace, test.query))
-			err = writeResp(csvWriter, test.user, curlURL, jsonFilePath, jsonResp)
-			require.NoError(s.T(), err)
+			if !downStreamClusterRegex.MatchString(curlURL) {
```

Comment:
@cmurphy I decided to put this check, so when the test is ran on a downstream cluster it doesn't write to the JSON, that way our drone test run does have a huge git diff at the end. What do you think?

Reply:
That works for now. fyi this is what I'm working on to have it skip the downstream

https://github.com/rancher/rancher/pull/40536/files#diff-3366c0a89c53a1d567a3c9a95378e6723ffba144299511dd1f6c0907376c75e2R266
https://github.com/rancher/rancher/pull/40536/files#diff-3366c0a89c53a1d567a3c9a95378e6723ffba144299511dd1f6c0907376c75e2R1686-R1694

Reply:
Ok sounds good, thanks for the review!